.github/workflows/matrix.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Matrix
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
```

Comment:
This might be overkill, but right now I've set it up to run the test matrix on pushes to develop/master + on any pull request opened to either of the same branches. So it'll run on your PR + again after merge. I'm open to changing this if anyone thinks we should.